app/services/api/v1/staffs/sync_service.rb

```diff
@@ -49,16 +50,11 @@ def school_staffs
           school.staffs.all.includes(:standards)
         end
 
-        def attendances
-          attendances = Attendance.where("date BETWEEN ? AND ?", DateTime.now.beginning_of_month, DateTime.now.end_of_day).all
-          attendances.each_with_object({}) do |c, h|
-            (h[c.standard_id] ||= []).push(
-              id:         c.id,
-              date:       c.date,
-              present:    c.present,
-              student_id: c.student_id,
-              sms_sent:   c.sms_sent
-            )
+        def find_attendances
+          @attendances = {}
+          ::Attendance.where("date BETWEEN ? AND ?", DateTime.now.beginning_of_month, DateTime.now.end_of_day)
+                      .group_by(&:standard_id).each do |standard_id, attendances|
+            @attendances[standard_id] = attendances.map(&:in_json)
```

Comment:
::Attendance.where("date BETWEEN ? AND ?", DateTime.now.beginning_of_month - 2.month, D
ateTime.now.end_of_day).select(:id, :standard_id, :date, :present, :student_id, :sms_sent).group_by(&:standard_
id).as_json()
Will this query work ? @pegasusflyhigh

Reply:
Yes it will work